content/blog/2018-06-08-unconf18_recap_3.md

```diff
@@ -0,0 +1,81 @@
+---
+slug: "unconf_recap_3"
+title: "Unconf18 projects 1: jobstatus, motifator, QcodeR, opencv, trackmd"
+date: 2018-06-08
+authors:
+  - name: Maëlle Salmon
+    url: http://www.masalmon.eu/
+categories: blog
+tags:
+- community
+- ropensci
+- meetings
+- unconf
+- unconf18
+---
+
+Following other [unconf18 recaps](https://ropensci.org/tags/unconf18/), here is an overview of five other projects. In the following weeks, a handful of unconf18 groups will share more thorough posts about their work.
+
+In the spirit of exploration and experimentation at rOpenSci unconferences, these projects are not necessarily finished products or in scope for rOpenSci packages.
+
+Let's dive into today's 5 projects in focus! 
+### `jobstatus`
+
+![jobstatus in action](/img/blog-images/2018-06-08-unconf18_recap_3/jobstatus.gif)
+
+**Summary:** `jobstatus` helps keep an eye on how complex and long-running jobs, including jobs running in parallel, are progressing. The maintainers won’t have much time to extend and improve the package so they'd love to have help: if you’re keen please let them know in the [issues tracker](https://github.com/ropenscilabs/jobstatus/issues)!
+
+**Team:** [Nick Golding](https://github.com/goldingn), [Mark Padgham](https://github.com/mpadge), [Joe Cheng](https://github.com/jcheng5).
+
+
+**Github:** https://github.com/ropenscilabs/jobstatus
+
+![jobstatus team](/img/blog-images/2018-06-08-unconf18_recap_3/jobstatus.jpg)
+
+`jobstatus` *team picture by [Mauro Lepore](https://twitter.com/mauro_lepore).*
+
+### `motifator`
+
+**Summary:** `motifator` is a package aiming at generating spatial motifs that replicate the ones seen in real-world data, without including any potentially identifying information. Potential applications include the analysis of the impact of spatial clustering of different levels of vaccination coverage in a model of infectious disease transmission.
+
+**Team:** [Jon Zelner](http://www.jonzelner.net/)
+
```

Comment:
need a return between `Zelner` & `GitHub`. `</br>`?

Reply:
same in opencv & trackmd

Reply:
Wasn't able to fix cf https://github.com/ropensci/roweb2/issues/20